src/component/mxgraph/ShapeConfigurator.ts

```diff
@@ -14,14 +14,29 @@
  * limitations under the License.
  */
 import { MxGraphFactoryService } from '../../service/MxGraphFactoryService';
+import { mxgraph } from 'ts-mxgraph';
+import { ShapeBpmnElementKind } from '../../model/bpmn/shape/ShapeBpmnElementKind';
+import EndEventShape from './shape/EndEventShape';
 
-/* eslint-disable @typescript-eslint/no-explicit-any */
 export default class ShapeConfigurator {
-  private mxShape: any = MxGraphFactoryService.getMxGraphProperty('mxShape');
+  private mxClient: typeof mxgraph.mxClient = MxGraphFactoryService.getMxGraphProperty('mxClient');
+  private mxShape: typeof mxgraph.mxShape = MxGraphFactoryService.getMxGraphProperty('mxShape');
+  private mxCellRenderer: typeof mxgraph.mxCellRenderer = MxGraphFactoryService.getMxGraphProperty('mxCellRenderer');
 
-  public initMxShapePrototype(isFF: boolean): void {
+  public configureShapes(): void {
+    this.initMxShapePrototype(this.mxClient.IS_FF);
+    this.registerShapes();
+  }
+
+  private registerShapes(): void {
+    this.mxCellRenderer.registerShape(ShapeBpmnElementKind.EVENT_END, EndEventShape);
```

Comment:
I don't know what is better.

If we have 2 enum, it would be more difficult to refactor the declaration of the styles, with a loop on the ShapeBpmnElementKind, like we did in a converter.

Reply:
These are default shapes in mxGraph:
SHAPE_RECTANGLE   =    rectangle
SHAPE_ELLIPSE   =    ellipse
SHAPE_RHOMBUS   =    rhombus
SHAPE_CYLINDER    =    cylinder
SHAPE_CONNECTOR   =    connector
SHAPE_ACTOR   =    actor
SHAPE_TRIANGLE    =    triangle
SHAPE_HEXAGON   =    hexagon
SHAPE_CLOUD   =    cloud
SHAPE_LINE    =    line
SHAPE_ARROW   =    arrow
SHAPE_ARROW_CONNECTOR   =    arrowConnector
SHAPE_DOUBLE_ELLIPSE    =    doubleEllipse
SHAPE_SWIMLANE    =    swimlane
SHAPE_IMAGE   =    image
SHAPE_LABEL   =    label

so I think only label may be in conflict

Reply:
ok, let's keep that for now, we will be able to manage this later